README.md

```diff
@@ -1,33 +1,195 @@
+
+```python
+import desbordante
+
+TABLE = '../examples/datasets/theatres_mfd.csv'
+METRIC = 'euclidean'
+LHS_INDICES = [0]
+RHS_INDICES = [2]
+PARAMETER = 5
+
+algo = desbordante.MetricVerifier()
+algo.set_option('table', (TABLE, ',', True))
+algo.set_option('is_null_equal_null')
+algo.load_data()
+algo.set_option('lhs_indices', LHS_INDICES)
+algo.set_option('metric', METRIC)
+algo.set_option('parameter', PARAMETER)
+algo.set_option('dist_from_null_is_infinity')
+algo.set_option('rhs_indices', RHS_INDICES)
+algo.execute()
+if algo.mfd_holds():
+    print('MFD holds')
+else:
+    print('MFD not holds')
+```
+```text
+>>> MFD holds
+```
+4) Discover approximate functional dependencies with various error thresholds. Here, we showcase a preferred approach to configuring algorithm options. Furthermore, we are using a pandas dataframe to load data from a CSV file.
+```python-repl
+>>> import desbordante
+>>> import pandas as pd
+>>> pyro = desbordante.Pyro()
+>>> df = pd.read_csv('iris.csv', sep=',', header=0)
+>>> pyro.load_data(df)
+>>> pyro.execute(error=0.0)
+>>> pyro.get_fds()
+[( 0 1 2 ) -> 4, ( 0 2 3 ) -> 4, ( 0 1 3 ) -> 4, ( 1 2 3 ) -> 4]
+>>> pyro.execute(error=0.1)
+>>> pyro.get_fds()
+[( 2 ) -> 0, ( 2 ) -> 1, ( 0 ) -> 2, ( 2 ) -> 4, ( 2 ) -> 3, ( 3 ) -> 2, ( 3 ) -> 0, ( 0 ) -> 1, ( 0 ) -> 3, ( 1 ) -> 0, ( 1 ) -> 2, ( 3 ) -> 4, ( 3 ) -> 1, ( 1 ) -> 3, ( 0 ) -> 4, ( 1 ) -> 4]
+>>> pyro.execute(error=0.2)
+>>> pyro.get_fds()
+[( 2 ) -> 1, ( 2 ) -> 0, ( 2 ) -> 4, ( 0 ) -> 2, ( 2 ) -> 3, ( 0 ) -> 1, ( 3 ) -> 4, ( 3 ) -> 2, ( 3 ) -> 1, ( 3 ) -> 0, ( 1 ) -> 2, ( 0 ) -> 3, ( 0 ) -> 4, ( 1 ) -> 0, ( 1 ) -> 4, ( 1 ) -> 3]
+>>> pyro.execute(error=0.3)
+>>> pyro.get_fds()
+[( 2 ) -> 1, ( 0 ) -> 2, ( 2 ) -> 0, ( 3 ) -> 0, ( 2 ) -> 3, ( 1 ) -> 0, ( 2 ) -> 4, ( 3 ) -> 2, ( 0 ) -> 1, ( 1 ) -> 2, ( 3 ) -> 1, ( 3 ) -> 4, ( 0 ) -> 3, ( 4 ) -> 2, ( 4 ) -> 1, ( 0 ) -> 4, ( 1 ) -> 3, ( 1 ) -> 4, ( 4 ) -> 3]
+```
+
+## Web interface
+
+While the Python interface makes building interactive applications possible, Desbordante also offers a web interface which is aimed specifically for interactive tasks. For pattern discovery/validation a user may specify parameters, browse results, employ advanced visualizations and filters, all in a convenient way. Furthermore, we manually select the most interesting Python scenarios and implement them inside our web version. Currently, only the typo detection scenario is implemented.
+
+You can try the deployed web version [here](https://desbordante.unidata-platform.ru/). You have to register in order to process your own datasets. Keep in mind that due to a large demand various time and memory limits are enforced (and a task is killed if it goes outside the acceptable ranges). The source code of the web interface is kept in a separate [repo](https://github.com/vs9h/Desbordante).
```

Comment:
```suggestion
You can try the deployed web version [here](https://desbordante.unidata-platform.ru/). You have to register in order to process your own datasets. Keep in mind that due to large demand various time and memory limits are enforced. The source code of the web interface is kept in a separate [repo](https://github.com/vs9h/Desbordante).
```
I don't think this technicality is necessary for a potential user

Reply:
It is important, since there is like 30 sec timelimit. So, I expect many datasets to fail and therefore I believe it is important to mention it explicitly.

Reply:
"various time and memory limits are enforced: processing is aborted if either of the two exceed the limits."

Reply:
"various time and memory limits are enforced: processing is aborted if either of the two exceed them."